robottelo/ui/locators.py

```diff
@@ -764,7 +764,8 @@ def __iter__(self):
     # Hosts
 
     # host.primary
-    "host.new": (By.XPATH, "//a[contains(@href, '/hosts/new')]"),
+    "host.new": (By.XPATH, "//a[contains(@href, '/hosts/new') and "
```

Comment:
we'll probably have to go in later and switch the quotes types in this file. so no worries on these.